.github/workflows/release.yml

```diff
@@ -28,13 +28,15 @@ jobs:
           token: ${{ secrets.GH_REPO_TOKEN }}
 
   rebase:
+    name: 'Rebase develop onto main'
     uses: myparcelnl/prestashop/.github/workflows/rebase.yml@main
+    secrets: inherit
     needs: release
     if: always()
-    secrets: inherit
     with:
       base: 'main'
       target: 'develop'
+      force: true
```

Comment:
dit zou ik nog even weglaten, ik heb het er als "test" bij woo in gezet, om te kijken of het werkt en niks sloopt